app/models/article.js

```diff
@@ -0,0 +1,16 @@
+import DS from "ember-data";
+
+const { attr, belongsTo } = DS;
+
+export default DS.Model.extend({
+    title: attr(),
```

Comment:
Ok thanks for catching that. Also what are the comment decorators you used in user model?